include/grpcpp/impl/codegen/async_generic_service.h

```diff
@@ -39,7 +39,7 @@ class GenericServerContext final : public ServerContext {
   const grpc::string& host() const { return host_; }
 
  private:
-  friend class Server;
+  friend class grpc_impl::Server;
```

Comment:
Maybe having one

using grpc_impl::Server;

would eliminate the need of specifying grpc_impl:: everywhere

Reply:
`using` is forbidden in public headers :P

Reply:
Ah I see